CuBeanzQueryable.sol

```diff
@@ -0,0 +1,251 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+error NotOwner();
+error MaxTeamMints();
+error RoundSoldOut();
+error SaleNotStarted();
+error InvalidValue();
+error MaxMints();
+error ContractMint();
+/*
+
+░█████╗░██╗░░░██╗██████╗░███████╗░█████╗░███╗░░██╗███████╗
+██╔══██╗██║░░░██║██╔══██╗██╔════╝██╔══██╗████╗░██║╚════██║
+██║░░╚═╝██║░░░██║██████╦╝█████╗░░███████║██╔██╗██║░░███╔═╝
+██║░░██╗██║░░░██║██╔══██╗██╔══╝░░██╔══██║██║╚████║██╔══╝░░
+╚█████╔╝╚██████╔╝██████╦╝███████╗██║░░██║██║░╚███║███████╗
+░╚════╝░░╚═════╝░╚═════╝░╚══════╝╚═╝░░╚═╝╚═╝░░╚══╝╚══════╝
+
+Dev: Shimazu Bohoro
+*/
+
+
+import "erc721a/contracts/extensions/ERC721AQueryable.sol";
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
+
+
+
+contract CuBeanzQueryable is  ERC721AQueryable , Ownable, ReentrancyGuard {
+    using Strings for uint256;
+    using ECDSA for bytes32;
+
+    //@dev mint constraints
+    uint16 public teamMints;
+    uint16 maxTeamMints = 444;
+    uint16  roundOneSupply = 2222;
+    uint16  roundTwoSupply = 1778;
```

Comment:
Since there is no onlyOwner setter for: 
- `maxTeamMints` 
- `roundOneSupply` 
- `roundTwoSupply` 

these variables can be marked as constant to save gas.